README.md

```diff
@@ -73,55 +153,103 @@ USE_L10N = True
+2. Update your project's Django settings file with the settings required for using the Transifex Python toolkit. Read more about [configuring your project](#configuration).
+3. [Add translation hooks in your templates](#add-translation-hooks)
+4. [Push the source content to Transifex](#push-source-content-to-transifex)
+5. [Translate content on Transifex](#translate-content-on-transifex)
+6. [Display translated content to your users](#display-translated-content)
+
+##### Add Translation Hooks
 
-#### 1. Add translation hooks
+###### Django template simple example 
 
 Open a Django template file (e.g. an `.html` file) and add the following:
-```
-{% load transifex %}
 
+```django
+{% load transifex %}
 <p>{% t "Hello!" %}</p>
 <p>{% t "I want to be translated." %}</p>
 ```
 
-#### 2. Push source content to Transifex
+###### Django view simple example 
 
-This command will collect all translatable strings and push them to Transifex.
+```django
+from transifex.native.django import t
+from django.http import HttpResponse
+
+def my_view(request):
+    output = t("Welcome aboard!")
+    return HttpResponse(output)
 ```
+	
+To see more examples on translation hooks checkout [Use in Templates and Views](#use-in-templates-and-views).	
+
+##### Push Source Content to Transifex
+
+This command will collect all translatable strings and push them to Transifex.
+
+```bash
 ./manage.py transifex push
 ```
 
-#### 3. Translate content on Transifex
+##### Translate Content on Transifex
 
 The next step is for your translators to translate the strings in various languages using Transifex.
-When a translation is added on Transifex, it becomes available over-the-air on your app.
-Please note that it can take a few minutes for the translations to become available on your app.
+When a translation is added on Transifex, it becomes available over-the-air on your application.
+Please note that it can take a few minutes for the translations to become available on your application.
 
-#### 4. Display translated content
+##### Display Translated Content
 
 The Transifex Toolkit automatically displays translated content in the language currently selected in your Django project.
+In order to allow changing the current language, you will need a **language picker**.
 
-In order to allow changing the current language, you will need the following:
+Here is an example of how you can add a **language picker** in your application. 
 
-##### 4.1 A language picker
+###### Project preparation for localization
 
-Here is an example of how you can add a language picker in your app.
-You can add this on the same HTML file you added the translatable strings before, like so:
+Please follow this steps in order to prepare your Django project for localization:
+
+1. You can add this on the same HTML file you added the translatable strings before, like so:
+
+```html
+   <form>
+   <!-- ... -!>
+```
```

Comment:
If you prepend each line in this snippet with 3 spaces, it will appear as part of the first bullet:

```
1. first bullet

   ` ``
   first bullet code quote
   ` ``

2. second bullet

   ` ``
   second bullet code quote
   ` ``
```

vs

```
1. first bullet

` ``
first bullet code quote
` ``

2. second bullet

` ``
second bullet code quote
` ``
```

The same applies to the next bullet as well.

_Note: I used `<backtick><space><backtick><backtick>` instead of `<backtick><backtick><backtick>` in my example so that the rendered version of my comment would work._

Reply:
Done